yt_dlp/extractor/livestreamfails.py

```diff
@@ -0,0 +1,50 @@
+from .common import InfoExtractor
+from ..utils import traverse_obj
+import json
+import time
+import calendar
+
+
+class LivestreamfailsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?livestreamfails\.com/clip/(?P<id>[0-9]+)'
+    _TESTS = [{
+        'url': 'https://livestreamfails.com/clip/139200',
+        'md5': '8a03aea1a46e94a05af6410337463102',
+        'info_dict': {
+            'id': '139200',
+            'ext': 'mp4',
+            'display_id': 'ConcernedLitigiousSalmonPeteZaroll-O8yo9W2L8OZEKhV2',
+            'title': 'Streamer jumps off a trampoline at full speed',
+            'creator': 'paradeev1ch',
+            'thumbnail': 'https://livestreamfails-image-prod.b-cdn.net/image/3877b1d38db083fa25c82685bbaf645637e575ea.png',
```

Comment:
better to use regex here

Reply:
To what extent? The [example](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-support-for-a-new-site) has regex for the thumbnail but not the title.

Reply:
this review points to `thumbnail` key

Reply:
My guess is that this is because the image should be able to change URL as long as it is still results in a 200. Is that correct?

Reply:
Yes. If you know the URL doesn't change over time, you can keep the current test. Otherwise, use a regex to future-proof